newproject01/newproject01/q2.c

```diff
@@ -0,0 +1,39 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <string.h>
+#define N 51
+char letterAtIndex(char*, int);
+main2()
+{
+	char string[N], returnChar;
+	int m;
+
+	printf("Enter string(not more than 50 chars), please: ");
```

Comment:
replace the magic number to a parameter, what if you would like to change the size from 50?